scripts/shared/clusters.sh

```diff
@@ -121,8 +123,28 @@ function deploy_weave_cni(){
 }
 
 function deploy_kind_ovn(){
-    echo "OVN on kind currently not supported"
-    exit 1
+    export K8s_VERSION=${version}
+    export NET_CIDR_IPV4="${cluster_CIDRs[${cluster}]}"
+    export SVC_CIDR_IPV4="${service_CIDRs[${cluster}]}"
+    export KIND_CLUSTER_NAME="${cluster}"
+    export OVN_IMAGE="quay.io/vthapar/ovn-daemonset-f:latest"
```

Comment:
I guess you're planning to change it later to one we will build?

Reply:
Yes. Currently running into some issues trying to build from our shipyard base image.